### PR TITLE
Fixed TypeError during running ``ralph ldap_sync``

### PR DESCRIPTION
**Pull request by ar4s**
Related to #2298

### CODE REVIEW

File: src/ralph/accounts/management/commands/ldap_sync.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def get_nested_groups():
     users_groups = defaultdict(set)
     nested_groups = getattr(settings, 'AUTH_LDAP_NESTED_GROUPS', None)
     if not nested_groups:
-        return
+        return None, None
```
Comment (mkurek) [inline src/ralph/accounts/management/commands/ldap_sync.py]: maybe it'll be better to return here `return group_users, users_groups`? Then it'll have types of the valid result

**Comment by mkurek** [comment]
:ok:

**Comment by ghost** [comment]
:ok:

**Comment by szok** [comment]
:ok:

**Pull request merged**